docs/articles/analyzers/core.md

```diff
@@ -71,3 +79,24 @@ public class GetSpecificGuilds()
     }
 }
 ```
+
+## Usage Warning DSP0008
+
+Use `DiscordPermissions` and its operators isntead of doing raw operations on `DiscordPermission`.
+
+DiscordPermission relies on using `long`. Discord can have permissions that is bigger than what a long can store.  
+`DiscordPermission` can overflow in these scenarios, `DiscordPermissions` is a class that does not overflow and can
+handle these scenarios.
+Prefer using `DiscordPermissions` to prevent issues with integer overflowing.
+
+The following sample will generate DSP0008
```

Comment:
this is incorrect. `DiscordPermission` is an enum containing the bit offsets of each flag (which is an implementation detail and should never be relied on). it's never correct to rely on `DiscordPermission` to represent permissions because it fundamentally doesn't represent permissions; not just when it would overflow an `ulong`.